Source/WebKit/Configurations/GPUExtension.xcconfig

```diff
@@ -25,6 +25,7 @@
 
 INFOPLIST_FILE = Shared/AuxiliaryProcessExtensions/GPUExtension-Info.plist;
 INFOPLIST_KEY_CFBundleDisplayName = GPUExtension;
-PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.GPUExtension;
-PRODUCT_BUNDLE_EXECUTABLE = GPUExtension;
+PRODUCT_NAME = com.apple.WebKit.GPU;
```

Comment:
Does this actually work? The binary will still end in `.appex`, so it's a different name than the XPC services.

Would it instead be sufficient to change the process name at runtime to match the status quo names, similar to what we do in `WebProcess::updateProcessName`?

Reply:
Yes, I verified that this works. However, I have reduced the patch to just change the bundle ID, since that is the main objective of the patch.

Thanks for reviewing!